Code/UnitTests/CoreTest/Scripting/TypeScriptTest.cpp

```diff
@@ -92,7 +92,11 @@ EZ_CREATE_SIMPLE_TEST(Scripting, TypeScript)
 
     ezStringBuilder sTestDataDir(">sdk/", ezTestFramework::GetInstance()->GetRelTestDataPath());
     sTestDataDir.AppendPath("Scripting/TypeScript");
-    if (!EZ_TEST_RESULT(ezFileSystem::AddDataDirectory(sTestDataDir, "TypeScriptTest", "TypeScriptTest", ezFileSystem::AllowWrites)))
+    if (!EZ_TEST_RESULT(ezFileSystem::AddDataDirectory(sTestDataDir, "TypeScriptTest")))
+      return;
+
+    const ezStringBuilder sWriteDir = ezTestFramework::GetInstance()->GetAbsOutputPath();
```

Comment:
Why does this test need this directory now and not before?

Reply:
The old code was mapping the `TypeScriptTest` folder from the SDK as both the source dir for the test as well as the target dir for the test output. As this is part of the Data folder, it will be read-only in the apk. Thus, I need to map two directories instead, one for reading the test data and one for writing the temp files the test generates.